stickyheaders/src/main/java/com/brandongogetap/stickyheaders/StickyLayoutManager.java

```diff
@@ -170,4 +170,13 @@ private void cacheHeaderPositions() {
             positioner.setHeaderPositions(headerPositions);
         }
     }
+
+    /**
+     * We override this method in order to prevent a crash.
+     * https://stackoverflow.com/questions/30220771/recyclerview-inconsistency-detected-invalid-item-position?page=1&tab=votes#tab-top
+     */
+    @Override
+    public boolean supportsPredictiveItemAnimations() {
+        return false;
```

Comment:
Isn't this the default? 

https://developer.android.com/reference/android/support/v7/widget/RecyclerView.LayoutManager.html#supportsPredictiveItemAnimations()